### DISCUSSION TITLE
Cancel workflow is not working

### DISCUSSION THREAD

**Discussion by shannonjarcher** [community/community#49085] [Actions]
### Select Topic Area

Bug

### Body

Related to https://github.com/orgs/community/discussions/25570 

Attempts to Cancel workflow show an error "Failed to cancel workflow". I've tried multiple times over the last 12 hours to cancel the workflow so I can rerun it as it is my production deploy step.

There was an incident with the Actions api that started all this but it has since been marked resolved while issues with Actions are ongoing.

**Comment by jsoref** [comment]
Try opening a ticket with https://support.github.com/ ?
